package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this file is not required
delete this file 
and while pushing first pull the changes